lib/order.rb

```diff
@@ -0,0 +1,77 @@
+require_relative 'menu'
+require_relative 'text_message'
+
+class Order
+
+  attr_reader :basket, :menu
+
+  def initialize(menu = Menu.new)
+    @basket = Hash.new(0)
+    @menu = menu
+  end
+
+  def add_dish(item)
```

Comment:
Maybe this method could take an optional 2nd argument so you can add more than one of the item at a time
Something along the lines of `def add_dish(item, quantity = 1)`

Reply:
Yes I totally agree! I thought about it for a bit but eventually decided to keep it simple. Next time I'll do as you suggest :) Thanks!